config/lava/chromeos/tast.jinja2

```diff
@@ -0,0 +1,32 @@
+- test:
+      timeout:
+        minutes: 45
+      docker:
+        image: {{ docker_image }}
+        volumes:
+          - /tmp/octopus:/home/cros-tast/octopus
+        wait:
+          device: true
+      results:
+          location: /home/cros-tast/lava
+      definitions:
+      - repository:
+          metadata:
+            format: Lava-Test Test Definition 1.0
+            name: docker-ping
+          run:
+            steps:
+              - 'cd /home/cros-tast'
+{%- if cros_tast_tarball %}
+              - 'curl -s {{cros_tast_tarball}} | tar xjf -'
+{% endif %}
+{%- if rootfs_url %}
+              - 'curl -s {{rootfs_url}}tast.tgz | tar xzf -'
+              - 'cp remote_test_runner /usr/bin/remote_test_runner'
+{% endif %}
```

Comment:
And if we're always using the tarball URL then we should be able to drop this too I believe.

Reply:
Sent PR https://github.com/kernelci/kernelci-core/pull/1083